api/foxpass_remove_user_groups.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+"""
+This script requires the external libraries from requests
+pip install requests
+
+To run:
+python foxpass_user_cleanup.py --api-key <api_key>
```

Comment:
Pls fix the script name. Also, a small description what this script is doing will be useful.